README.md

```diff
@@ -116,3 +116,24 @@
 - [ ] Deploy services with HA: PostgreSQL, RabbitMQ, etcd
 - [ ] Add monitoring to the pipeline (rovers, borges and associated services)
 - [ ] Migrate staging cluster to the new deployment method
+
+## Being a better company to work at
+
+### Successfully hire for each open role
+
+- [ ] Hire a Lead engineer for the Applications team 
+- [ ] Hire a Senior test engineer 
+- [ ] Hire two interns in Q4 for Q1-2 for ML team
+- [ ] Attract relevant profiles through our community
+- [ ] Add feedback after every interview on Lever
```

Comment:
@gomesfernanda , @marnovo it's coming from the previous quarter, not sure what's the state of the survey so let me know if I should keep it or no. I'm not sure what's the current state of the project.